CRPPL.g4

```diff
@@ -15,9 +15,9 @@ validexpr: generalquery
          ;
 
 //PARSER RULES.
-generalquery: RESERVEDWORD (OPERATOR RESERVEDWORD)? LITERAL (SEPARATOR LITERAL)* (RESERVEDWORD LITERAL OPERATOR LITERAL (SEPARATOR LITERAL OPERATOR LITERAL)*)?;
+generalquery: GET (OPERATING_FUNCTION OF)? LITERAL (COMMA GET (OPERATOR OF)? LITERAL)* (FOR LITERAL OPERATOR LITERAL (COMMA LITERAL OPERATOR LITERAL)*)?;
```

Comment:
Mond, are we gonna support literals only for the general query? Or we can support identifiers as well?
Ex: 
get sum of **'revenue'** for 'year' = '2021', 'month' = '11'

create_function do_graph_inside(r**evenue**)
	get sum of **revenue** for 'year' = '2021', 'month' = '11'

if so, perhaps we can allow literals in the general query